src/api/bundles.rs

```diff
@@ -36,14 +78,22 @@ impl BundlesCommand {
 }
 
 #[derive(Parser, Debug)]
-
 pub struct CreateCommand {
-    /// The PRNs of the artifact versions to include binaries from in the bundle.
+    /// The PRNs of the artifact versions to include binaries from in the bundle (API v1 only).
     ///
     /// Values can be provided by passing each value in a flag
     /// or by delimiting all values with ","
     #[clap(short, long, value_parser, num_args = 1.., value_delimiter = ',')]
-    artifact_version_prns: Vec<String>,
+    artifact_version_prns: Option<Vec<String>>,
+
+    /// The binaries to include in the bundle (API v2 only).
```

Comment:
I believe the help text here is incorrectly implying `artifact_version` PRNs, rather than binary PRNs.

eg, says `prn=prn:1:org:artifact_version:id` and I think it should be `prn=prn:1:org:binary:id`

Reply:
updated